packages/commerce/backend/src/open-payments/service.ts

```diff
@@ -155,9 +153,86 @@ export class OpenPayments implements IOpenPayments {
       throw new InternalServerError()
     })
 
+    await order.$query().patch({
+      quoteId: quote.id,
+      paymentPointerUrl: customerPaymentPointer.id,
+      continueToken: outgoingPaymentGrant.continue.access_token.value,
+      continueUri: outgoingPaymentGrant.continue.uri
+        .replace('localhost', 'rafiki-auth')
+        .replace('auth/', '')
```

Comment:
```suggestion
        .replace('auth/', '')
```

I think this is a mistake coming from Rafiki's auth. Waiting for Nathan's response.